recipes/scattertext/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "scattertext" %}
+{% set version = "0.0.2.9.8" %}
+{% set sha256 = "5fd562852a403f9f505aa4f4cbce83bf56714ad4f0cd05412232340cfc10698c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win32]
+  entry_points:
+    - scattertext = scattertext.CLI:main
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.0
```

Comment:
Yes: 

https://github.com/JasonKessler/scattertext/blob/master/setup.py#L31

Also, I'm still hoping to get a response from @JasonKessler about co-maintaining this recipe. Unfortunately the asshole that organized the Charlottesville march shares the same name and I'm guessing our good data scientist friend might have missed the original mention during all that.